### PR TITLE
Configure Nagios

### PR DESCRIPTION
**Pull request by SwannCroiset**
The patch configures both Nagios and Heka services:

* Nagios hosts and services are configured from mines.
* Heka services get nagios outputs to send alarms to Nagios,
  this concerns metric_collector and the aggregator.

### CODE REVIEW

File: classes/cluster/mk22_full_scale/stacklight/init.yml [hunk 1]
```diff
@@ -14,6 +14,14 @@ parameters:
...
     influxdb_user: lma
     influxdb_password: lmapass
+    nagios_host: ${_param:stacklight_monitor_address}
+    nagios_status_port: 8001
+    nagios_username: nagiosadmin
+    nagios_password: secret
+    nagios_notification_smtp_server: 127.0.0.1
+    nagios_notification_from: 'nagios@localhost'
+    nagios_notification_email: 'root@localhost'
+    nagios_default_host_alarm_clusters: 00-clusters
```
Comment (elemoine) [inline classes/cluster/mk22_full_scale/stacklight/init.yml]: The above two are already defined in the nagios system class. I think we can just use the defaults that are provided there, no?

File: classes/cluster/mk22_lab_advanced/stacklight/init.yml [hunk 2]
```diff
@@ -12,6 +12,14 @@ parameters:
...
     influxdb_user: lma
     influxdb_password: lmapass
+    nagios_host: ${_param:stacklight_monitor_address}
+    nagios_status_port: 8001
+    nagios_username: nagiosadmin
+    nagios_password: secret
+    nagios_notification_smtp_server: 127.0.0.1
+    nagios_notification_from: nagios@localhost
+    nagios_notification_email: root@localhost
+    nagios_default_host_alarm_clusters: 00-clusters
```
Comment (elemoine) [inline classes/cluster/mk22_lab_advanced/stacklight/init.yml]: Ditto.

File: classes/cluster/mk22_scale_mirantis/stacklight/init.yml [hunk 3]
```diff
@@ -15,6 +15,14 @@ parameters:
...
     influxdb_user: lma
     influxdb_password: lmapass
+    nagios_host: ${_param:stacklight_monitor_address}
+    nagios_status_port: 8001
+    nagios_username: nagiosadmin
+    nagios_password: secret
+    nagios_notification_smtp_server: 127.0.0.1
+    nagios_notification_from: 'nagios@localhost'
+    nagios_notification_email: 'root@localhost'
+    nagios_default_host_alarm_clusters: 00-clusters
```
Comment (elemoine) [inline classes/cluster/mk22_scale_mirantis/stacklight/init.yml]: Ditto.

File: classes/cluster/mk22_full_scale/stacklight/init.yml (hunk 1 above)
Comment (SwannCroiset) [inline classes/cluster/mk22_full_scale/stacklight/init.yml]: the idea was to have different values for each _lab_

Comment (elemoine) [inline classes/cluster/mk22_full_scale/stacklight/init.yml]: Ok then.

**Pull request merged**